dkim/header_test.go

```diff
@@ -102,3 +102,11 @@ func TestHeaderPicker_Pick(t *testing.T) {
 
 	})
 }
+
+func TestFoldHeaderField(t *testing.T) {
+	header := `DKIM-Signature: a=rsa-sha256; bh=bvW0aiEWdP0ie2rawBb+IiTxlHI9KgEIYjEYTGzMRa0=; c=simple/simple; d=test.xxxxxxxxxx.xxxxx; h=Return-Path:From:To:Subject:Message-ID:Date; s=brisbane; t=1583859773; v=1; b=IIlaZ79pLDdypX2YzJuy9EvxsbrqF360CLhkQqMqVC/GIa1K/X8p8POny7WuVuFTMHLvA1zH9YcXgEgIH+X0GgGsqqzq8ETL+QGZ9g1EKCmIg+ZEYV5JnH25Ar6bJpGra3jg6sIxKi6CthpK50kd8T0Q3K/oZGot4BkLaPqogpo=;`
```

Comment:
Can we add a comment among these lines:

> The header field length is a multiple of 75. See https://github.com/emersion/go-msgauth/issues/23

So that we don't forget about the reason why we have this test?

Reply:
Yeah! Maybe it is better to create a fake header with a minimum length that generates the issue! What do you think?